tests/publiccloud/mr_test_setup_env.pm

```diff
@@ -0,0 +1,62 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Set up public cloud instance testing environment for 'mr_test'
```

Comment:
Should this module be located in `tests/publiccloud/` or in `tests/sles4sap/publiccloud`?

Reply:
Good question. I had considered putting this file in "tests/publiccloud/" but it it is `mr_test` specific environment setup so I put it in "tests/sles4sap/publiccloud".